src/preferences/dialog/dlgprefmixer.cpp

```diff
@@ -39,6 +39,15 @@ const ConfigKey kLowEqFreqKey = ConfigKey(kMixerProfile, kLowEqFrequency);
 const ConfigKey kLowEqFreqPreciseKey =
         ConfigKey(kMixerProfile, QStringLiteral("LoEQFrequencyPrecise"));
 
+const ConfigKey kXfaderModeKey = ConfigKey(EngineXfader::kXfaderConfigKey,
+        QStringLiteral("xFaderMode"));
+const ConfigKey kXfaderCurveKey = ConfigKey(EngineXfader::kXfaderConfigKey,
+        QStringLiteral("xFaderCurve"));
+const ConfigKey kXfaderCalibrationKey = ConfigKey(EngineXfader::kXfaderConfigKey,
+        QStringLiteral("xFaderCalibration"));
+const ConfigKey kXfaderReverseKey = ConfigKey(EngineXfader::kXfaderConfigKey,
+        QStringLiteral("xFaderReverse"));
+
```

Comment:
this depends on the constants in EngineXfader to be initialized first, right? That order is undefined, so I'm afraid this break / is UB. @daschuer you probably know more about that.

Reply:
ping @daschauer
Can you lend us your insight here? I'm blocking this PR over my concern here right now.

Reply:
This will work, because the initialization order of variables is undefined, but order of the regions is defined. 

1. The text segment, this is "initalized" at compile time and copied to the RAM via the OS binary loader. That are constexpr and static const.    
2. Global variables in the bss. They are nomaly memcpyed() by from their initalization values or just zeroed out.  
3. Global Object constructors in undefined order.  
4. Main()  

Since EngineXfader::kXfaderConfigKey is 1. 

I have recently verifies that a QStringLiteral() initalized variable is a combination from 1. 2.  So it means it would work as well. But it is a little bit brittle, because it is not a constexpr.

Reply:
Okay if you say so, I'll trust you to have more expertise here. Can still you link me a doc for this? I'd like to read more detail on it.